thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -56,23 +56,23 @@ export default Ember.Component.extend({
           break;
 
         case ROOTCAUSE_METRICS_SORT_PROPERTY_DATASET:
-          output = metricUrns.map(urn => [datasets[urn], urn]).sort();
+          output = metricUrns.map(urn => [datasets[urn], metrics[urn], urn]).sort();
```

Comment:
It's unclear to me what `.sort()` is sorting by. Can we use lodash's `sortBy()` for sorting in this function, wherever it applies? https://lodash.com/docs/4.17.4#sortBy

Reply:
tuple sort in natural order. Lodash's sortBy() applies to objects with named keys